index.html

```diff
@@ -0,0 +1,198 @@
+      href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+      integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Haddis Technology</title>
+  </head>
+  <body>
+    <div id="container">
+      <div class="nav-bar">
+        <nav class="primary-nav">
+          <ul class="primary-nav">
+            <li>
+              <a href="#"><i class="fab fa-facebook"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fab fa-twitter"></i></a>
+            </li>
+            <li>
+              <a href="#" class="my-page"><p>MyPage</p></a>
+            </li>
+            <li>
+              <a href="#" class="logout log-pad"><p>Logout</p></a>
+            </li>
+          </ul>
+        </nav>
+        <nav class="secondary-nav">
+          <div class="banner secondary-nav">
+            <a href="index.html">
+              <img class="logo-img" src="images/eagle-logo.jpeg" alt="logo" />
+              <p class="comp-name">Haddis Technology</p>
+            </a>
+          </div>
+          <ul class="secondary-nav nav-menu menu-nav">
+            <li class="nav-item">
+              <a href="index.html" class="link-nav home-link">Home</a>
+            </li>
+            <li class="nav-item">
+              <a href="about.html" class="link-nav">About</a>
+            </li>
+            <li class="nav-item"><a href="" class="link-nav">Program</a></li>
+            <li class="nav-item"><a href="" class="link-nav">Join</a></li>
+            <li class="nav-item"><a href="" class="link-nav">Sponsor</a></li>
+            <li class="nav-item"><a href="" class="link-nav">News</a></li>
+            <li class="nav-item">
+              <a href="" class="confference">Confference</a>
+            </li>
+          </ul>
+        </nav>
+        <div class="hamburger" id="hamburger">
+          <span class="l-1 bar"></span>
+          <span class="l-2 bar"></span>
+          <span class="l-3 bar"></span>
+        </div>
+      </div>
+      <section class="headlines">
+        <div class="advert-txt">
+          <p>"The software of your thoughts"</p>
+          <h1>WE COME WITH SMART TECHNO PRODUCTS!</h1>
+        </div>
+        <div class="box-txt">
+          <p>
+            In simple words, technology is when we take the concepts of science
+            and transform them into gadgets and devices that are useful to us as
+            human beings. When we look around us, almost everything is a result
+            of or a form of technology.
+          </p>
+        </div>
+        <div class="address-txt">
+          <p><strong>Next Zoom conference(monday july 25/2022)</strong></p>
+          <p>@ BahirDar, Ethiopia</p>
+        </div>
+      </section>
+
+      <!-- Main Program -->
+
+      <section class="main-program">
+        <h2 class="main-pro">See The All Program</h2>
```

Comment:
- [ ] Please ensure that the line beneath the `See All Program` heading is orange in color and there is a slight gap between the text and the horizontal line
> Behance
![image](https://user-images.githubusercontent.com/90556221/181478868-e41ef570-5c82-48b2-813b-667873855fca.png)

> Yours
![image](https://user-images.githubusercontent.com/90556221/181478946-636c87bd-7eaf-4a1e-bb4f-2e7ecf80cc66.png)